diff -Naur a/emscripten.d.ts b/emscripten.d.ts
--- a/emscripten.d.ts	2018-08-01 21:18:55.861335817 +0200
+++ b/emscripten.d.ts	2018-08-01 21:22:19.471723849 +0200
@@ -22,8 +22,8 @@
     function ccall(ident: string, returnType: string, argTypes: string[], args: any[]): any;
     function cwrap(ident: string, returnType: string, argTypes: string[]): any;
 
-    function setValue(ptr: number, value: any, type: string, noSafe: boolean): void;
-    function getValue(ptr: number, type: string, noSafe: boolean): any;
+    function setValue(ptr: number, value: any, type: string, noSafe?: boolean): void;
+    function getValue(ptr: number, type: string, noSafe?: boolean): any;
 
     var ALLOC_NORMAL: number;
     var ALLOC_STACK: number;

